include/alt-dummy/aes_alt.h

```diff
@@ -0,0 +1,23 @@
+/* aes_alt.h with dummy types for MBEDTLS_AES_ALT */
```

Comment:
`alt-dummy` is only ever going to apply to 3.6, not to 4.0 or tf-psa-crypto. So I think it should stay in 3.6. (I would normally treat it as framework material, I'm making an exception solely because it's only ever going to apply to a single consuming branch.)